packages/api/src/command/medical/patient/get-adt-subscribers.ts

```diff
@@ -0,0 +1,69 @@
+import { AdtSubscriberData } from "@metriport/core/domain/patient-settings";
+import { capture } from "@metriport/core/util";
+import { out } from "@metriport/core/util/log";
+import { errorToString } from "@metriport/shared";
+import { Sequelize } from "sequelize";
+import { PatientModel } from "../../../models/medical/patient";
+import { PatientSettingsModel } from "../../../models/patient-settings";
+
+export async function getAdtSubscribers(targetStates: string[]): Promise<AdtSubscriberData[]> {
```

Comment:
I couldn't put my finger on it in the last review, so I'm sorry to do this to you, but i dont like the name AdtSubscribers LOL. They haven't subscribed to ADTs, we are using this as a flag to note that we want to subcribe to ADTs about them. Thats a nuanced difference and imo the more nuanced the difference the more important the naming difference can be.

I think these are AdtEnrollees, and instead of a subscription we have an enrollment. Yeah this means we need to add a migration to change the subscriptions column name.